### ISSUE TITLE
Add Google Analytics for Login.gov 

### ISSUE THREAD

**Issue by AGarcia-Clarity** [department-of-veterans-affairs/va.gov-team#30098]
## Issue Description
In integrating Login.gov to vets-website there will need to be an addition verification that Login.gov related auth/sign in events are being logged to Google Analytics.  Additionally, we will need to request access to Google Analytics to monitor these events as well as the integration of Sentry errors for Login.gov.

---
## Tasks
- [x] Request access to Google Analytics
- [ ] Verify events are being sent to Google Analytics
- [ ] Add/verify errors are being logged in Sentry

## Acceptance Criteria
- [ ] I can see Login.gov auth events in Google Analytics 
- [ ] I can see Login.gov errors in Sentry

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by Kwoly** [comment]
Access to Google Analytics provided to both, @AGarcia-Clarity and myself. Moving this to blocked as we cannot test the login event or sentry log until closer to completion

**Comment by nsoutouras** [comment]
MSI - Should be able to unblock tomorrow after chatting with support team.

**Comment by nsoutouras** [comment]
Sprint Planning - May be worth breaking into new ticket.
